modules/update-upgrading-cli.adoc

```diff
@@ -77,6 +77,12 @@ $ oc adm upgrade channel stable-{product-version}
 ====
 For production clusters, you must subscribe to a `stable-\*`, `eus-*`, or `fast-*` channel.
 ====
++
+[NOTE]
+====
+If you want to update your cluster to the next minor version, select a channel that corresponds to that minor version before beginning your update.
+If you cannot see an update path to the target minor version, keep updating your cluster to the latest patch release for your current version until the next minor version is available in the path.
```

Comment:
Two nits with the current wording:

* "If you want to update your cluster" has an implicit "soon", at least for me.  But folks are free to set their channels long before they actually initiate the update, and [OCPBUGS-5469](https://issues.redhat.com/browse/OCPBUGS-5469) talks through why it sometimes takes a while for the cluster-version operator to evaluate conditional updates after a channel change.
* This line doesn't explicitly recommend leaving the channel on the next minor while updating within the current minor.

One way to get at both of those might be something like this:

> Of the channels available to your cluster, select the one closest to your desired semantics and target minor (4.y) version, whether or not you plan on updating in the near future.  The sooner the cluster understands what you want, the more effectively it can recommend plans to get you there.  Subsequent updates may be made at a time of your choosing within your previously-selected channel, even though it may take multiple updates to reach your target minor version.

Or some other wording that tries to clearly separate "pick a channel" and "initiate updates" into separate time-boxes.

Reply:
Text suggestion :

When you are ready to move to the next minor version select the channel that corresponds to that minor version. The sooner the cluster understands what you want, the more effectively it can recommend plans to get you there. Cluster might take some time to evaluate all the possible upgrades available and recommend the best possible upgrade recommendations to choose from.  The upgrade recommendation can change time to time as it is based on what is upgrade options available at that point of time. 

If you cannot see an update path to the target minor version, keep updating your cluster to the latest patch release for your current version until the next minor version is available in the path.